python/TestHarness/TestHarness.py

```diff
@@ -370,6 +372,7 @@ def testOutputAndFinish(self, tester, retcode, output, start=0, end=0):
       (reason, output) = self.buildPBSBatch(output, tester)
     elif self.options.dry_run:
       reason = 'DRY_RUN'
+      test['status_message'] = reason
```

Comment:
I pretty sure you don't need this line at all. You are setting a test "InputParameter" after the Tester has already finished (We're in the routine that deals with the results). I don't think you use it all so just delete.